InterviewFlashCards/QuestionsViewController.swift

```diff
@@ -120,7 +111,7 @@ class QuestionsViewController: UIViewController {
         nextButton.isHidden = false
```

Comment:
there was a previous comment asking to make `questionImages` strongly typed as `[UIImage]`